docs/dev/run_the_etl.rst

```diff
@@ -163,7 +163,7 @@ Running the ETL with Dagit
 Dagster needs a directory to store run logs and some interim assets. We don't
 distribute these outputs, so we want to store them separately from
 ``PUDL_OUTPUT``. Create a new directory outside of the pudl respository
-directory called ``dagster_home/``. Then set the ``DAGSTER_HOME`` environment
+directory called ``dagster_home/``, and create a ``dagster_home/dir`` subdirectoy. Then set the ``DAGSTER_HOME`` environment
```

Comment:
What was the issue that made you do this? It's working for me without this subdir, but I want to know what weirdness popped up in your setup!

Reply:
For me , if I remove this subir, when I run
`dagit -m pudl.etl -m pudl.ferc_to_sqlite`
i get
` File "/home/apptrain/mambaforge/envs/pudl-dev/lib/python3.11/site-packages/dagster/_core/instance/__init__.py", line 508, in get
    raise DagsterInvariantViolationError(
dagster._core.errors.DagsterInvariantViolationError: $DAGSTER_HOME "/home/apptrain/dagster_home/dir" is not a directory or does not exist. Dagster requires this environment variable to be set to an existing directory in your filesystem`

`(pudl-dev) apptrain@greenmachine:~/pudl$ dagit --version
dagit, version 1.3.5
(pudl-dev) apptrain@greenmachine:~/pudl$ python --version
Python 3.11.3
(pudl-dev) apptrain@greenmachine:~/pudl$`

Reply:
Ah, I think the root cause here is actually an earlier typo - If you set `DAGSTER_HOME` to the path to the `dagster_home` directory, not `dagster_home/dir`, it should work without adding the `dir` subdirectory.

Here's what I get:

```fish
> echo $DAGSTER_HOME                                              (pudl-dev) 
/Users/dazhong-catalyst/work/dagster_home
> ls $DAGSTER_HOME                                                (pudl-dev) 
dagster.yaml  history/      schedules/    storage/
```

And I suspect you'd get

```fish
> echo $DAGSTER_HOME                                              (pudl-dev) 
.../dagster_home/dir
> ls $DAGSTER_HOME                                                (pudl-dev) 
dagster.yaml  history/      schedules/    storage/
```

If that's true, I think we should just clarify the env var value instead of adding a new directory!

Reply:
@AppTrain - just checking in - if you confirm that the above works for you, we can update the docs changes accordingly!

Reply:
@jdangerx , yes, thanks for the reminder. I just confirmed, your assumptions are correct.  My DAGSTER_HOME included that 'dir' subdir.  I fix my ENV variable with "export DAGSTER_HOME=/home/apptrain/dagster_home" and  I'm able to run dagster without that 'dir' directory now. So I agree , we should fix the EXPORT line.